services/src/atdd-staging/src/main/java/org/openkilda/atdd/staging/steps/HealthCheckSteps.java

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2018 Telstra Open Source
```

Comment:
2018-2019

Reply:
the same issue in all new created  files

Reply:
Fixed.